openstack/barbican/values.yaml

```diff
@@ -15,6 +15,13 @@ global:
 pod:
   replicas:
     api: 2
+  resources:
+    requests:
+      memory: 24Mi
+      cpu: 10m
+    limits:
```

Comment:
please don't specify limits, for the barbican service qos-class burstable is enough (not guaranteed)